src/targz_sizes.c

```diff
@@ -120,8 +126,24 @@ int main(int argc, char** argv) {
         }
     }
 
+    unsigned int max_filename_blocks;
+    if ( LEVEL_IS_TEST( verbosity ) ) {
+        // when targz_sizes is in *test* mode, set max_filename_blocks
+        // very small (2).  This makes it easier to test the case where
+        // a long filename would overrun the buffer
+        max_filename_blocks=2;
```

Comment:
Formatting: put a single space before and after the assignment operator to make it consistent with the rest of this file.